### PR TITLE
Update project to cf-layout v4.7.0

### PR DESCRIPTION
**Pull request by jimmynotjim**
Wells have been released in Capital Framework. Updated the cf-layout dependency and made the necessary changes to include them in this project. 

## Removals

- Removed the cfgov-refresh well implementation.
- Removed `__flush, __full, and __stacked` well modifiers.

## Changes

- Moved the homepage specific well modifiers to `home.less`
- Converted the FWB questions to one large well with horizontal rules between questions.

## Testing

1. run `npm install`
1. run `gulp build`
1. navigate to a few pages with Wells (urls listed with screenshots below)

## Screenshots

http://localhost:8000/consumer-tools/everyone-has-a-story/ana-debts-you-dont-owe/

screen shot 2017-12-06 at 4 20 22 pm

http://localhost:8000/es/prepararse-para-comprar-casa/hipotecas/

screen shot 2017-12-06 at 4 23 06 pm

http://localhost:8000/policy-compliance/amicus/

screen shot 2017-12-06 at 4 19 43 pm

http://localhost:8000/es/prepararse-para-comprar-casa/credito/

screen shot 2017-12-06 at 4 23 34 pm

http://localhost:8000/consumer-tools/debt-collection/

screen shot 2017-12-06 at 4 23 58 pm

http://localhost:8000/

screen shot 2017-12-06 at 5 17 49 pm

http://localhost:8000/

screen shot 2017-12-06 at 5 27 34 pm

http://localhost:8000/consumer-tools/financial-well-being/

screen shot 2017-12-08 at 5 05 12 pm

http://localhost:8000/policy-compliance/know-you-owe-mortgages/settlement-professionals-guide/once-you-are-selected-settlement-agent/

screen shot 2017-12-08 at 5 33 26 pm

## Notes

- Wells have a max-width of 8 columns, note that when a well is placed in the main content area of a right-hand content template, the well does not expand the entire width of the main content.
- You may notice an odd extra bit of space above the well on http://localhost:8000/data-research/credit-card-data/college-credit-card-agreement/. I've logged it and hopefully it will be fixed when the page is republished.

## Checklist

* [x] PR has an informative and human-readable title
* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
* [x] Passes all existing automated tests
* [x] Any *change* in functionality is tested
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices
* [x] Project documentation has been updated
* [x] Reviewers requested with the [Reviewer tool](https://help.github.com/articles/about-pull-request-reviews/) :arrow_right:

### CODE REVIEW

**Comment by anselmbradford** [comment]
`gulp build`, right?

**Comment by jimmynotjim** [comment]
@anselmbradford whoops, sorry, updated the instructions

**Comment by anselmbradford** [comment]
Are tables too narrow? Is this right-alignment expected? (http://localhost:8000/data-research/credit-card-data/college-credit-card-agreement/)

screen shot 2017-12-11 at 10 14 22 am

**Comment by jimmynotjim** [comment]
@anselmbradford tables are only as wide as their content within them unless someone sets a specific width in the admin. It's kind of an odd thing but when I brought it up some time ago it was agreed that we can't default to 100%  width because there are existing tables that aren't very wide.

**Comment by anselmbradford** [comment]
Oh that's right, I recall the width setting in Wagtail now.

File: cfgov/unprocessed/css/pages/home.less [hunk 1]
```diff
@@ -17,6 +17,14 @@
 }
 
 .o-well__home {
+    max-width: 1230px;
```
Comment (Scotchester) [inline cfgov/unprocessed/css/pages/home.less]: This should be set to `@grid_wrapper-width`, I think.

File: cfgov/unprocessed/css/enhancements/forms.less [hunk 2]
```diff
@@ -337,6 +343,7 @@ input {
             box-sizing: border-box;
             padding-right: unit( 15px / @base-font-size-px, em );
             width: percentage( 1 / ( @scale-point-count + 1 ) );
```
Comment (Scotchester) [inline cfgov/unprocessed/css/enhancements/forms.less]: We should change how this is calculated. Losing the extra space that the previous flush treatment afforded is causing this to break onto 3 lines now. I think I'd probably just do something simple like 20% for this and 80% for `.o-scale_answer-options` below.

File: cfgov/wellbeing/jinja2/wellbeing/home.html [hunk 3]
```diff
@@ -850,82 +834,86 @@ <h3 class="a-legend">
...
-                                       data-gtm-category="Financial Well-Being Tool Interaction">
-                                <label class="a-label" for="read-to-me">
-                                    I read the questions to someone else and recorded their answers
-                                </label>
-                            </div>
-                        </li>
-                    </ul>
-                </fieldset>
+            <div class="o-well">
+                <div class="o-form_group">
```
Comment (Scotchester) [inline cfgov/wellbeing/jinja2/wellbeing/home.html]: I am not enthused about adding another wrapper here just to get some margin. Could we add this bottom margin to all `o-form_fieldsets`, or are there places where we don't want it? Failing that, could we just use a margin utility class on the fieldset?

File: cfgov/unprocessed/css/pages/home.less (hunk 1 above)
Comment (jimmynotjim) [reply cfgov/unprocessed/css/pages/home.less]: Yep, good catch.

File: cfgov/wellbeing/jinja2/wellbeing/home.html [hunk 4]
```diff
@@ -771,13 +757,11 @@ <h3 class="a-legend">
...
                 </fieldset>
-            </div>
-            <div class="o-well o-well__flush o-well__stacked o-scale">
-                <fieldset class="o-form_fieldset" id="question_10">
+                <fieldset class="o-form_fieldset o-scale" id="question_10">
                     <h3 class="a-legend">
                         My finances control my life
                     </h3>
-                    <div class="o-scale_answer-prefix">This statement applies to me</div>
+                    <p class="o-scale_answer-prefix">This statement applies to me</p>
```
[the remark this answers is not in the document]
Comment (jimmynotjim) [reply cfgov/wellbeing/jinja2/wellbeing/home.html]: For default paragraph spacing. While pairing with @amycesal she pointed out it was missing.

File: cfgov/wellbeing/jinja2/wellbeing/home.html (hunk 3 above)
Comment (jimmynotjim) [reply cfgov/wellbeing/jinja2/wellbeing/home.html]: There are places we don't want it and the groups have been the default layout elsewhere. I'd rather stick to consistency and tackle that in a new issue.

File: cfgov/unprocessed/css/enhancements/forms.less (hunk 2 above)
Comment (jimmynotjim) [reply cfgov/unprocessed/css/enhancements/forms.less]: Can you explain what you mean by "break onto 3 lines now", I think I'm missing something.

Comment (jimmynotjim) [reply cfgov/unprocessed/css/enhancements/forms.less]: Oh wait, I see what you mean.

File: cfgov/wellbeing/jinja2/wellbeing/home.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (Scotchester) [reply cfgov/wellbeing/jinja2/wellbeing/home.html]: What kind of spacing? Bottom margin when it's stacked over the answers?

File: cfgov/wellbeing/jinja2/wellbeing/home.html (hunk 3 above)
Comment (Scotchester) [reply cfgov/wellbeing/jinja2/wellbeing/home.html]: Fine for now, but I really want to kill `o-form_group`. It really shouldn't be necessary.

Comment (jimmynotjim) [reply cfgov/wellbeing/jinja2/wellbeing/home.html]: @Scotchester PRs always accepted 😛

File: cfgov/wellbeing/jinja2/wellbeing/home.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (jimmynotjim) [reply cfgov/wellbeing/jinja2/wellbeing/home.html]: Yes, didn't seem worth writing specific rules for something we already have built in.

**Pull request merged**